src/test/java/hudson/plugins/git/GitChangeSetTruncateTest.java

```diff
@@ -148,6 +148,12 @@ private void createFile(String path, String content) throws Exception {
     @Test
     @Issue("JENKINS-29977") // CLI git truncates first line of commit message in Changes page
     public void summaryTruncatedAtLastWord72CharactersOrLess() throws Exception {
+        System.out.println(" ---------------- ");
+        System.out.println(String.format(" commit summary '%s' (%d)", commitSummary, commitSummary.length()));
+        System.out.println(String.format(" msg            '%s' (%d)", changeSet.getMsg(), changeSet.getMsg().length()));
+        System.out.println(String.format(" expected       '%s' (%d)", expectedSummary, expectedSummary.length()));
```

Comment:
These debugging print statements need to be removed before the merge to master branch.  When the assertion fails, the assertion message includes msg and expected along with their values.